File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/accessing-a-form-region-at-run-time_3.vb
    Friend Sub Access_All_Form_Regions()
        Dim formRegion As Microsoft.Office.Tools.Outlook.IFormRegion
        For Each formRegion In Globals.FormRegions
            If TypeOf formRegion Is FormRegion1 Then
                Dim formRegion1 As FormRegion1 = _
                    CType(formRegion, FormRegion1)
                formRegion1.textBox1.Text = "Hello World"
            End If
        Next formRegion
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/accessing-a-form-region-at-run-time_3.cs
        internal void Access_All_Form_Regions()
        {
            foreach (Microsoft.Office.Tools.Outlook.IFormRegion formRegion 
                in Globals.FormRegions)
            {
                if (formRegion is FormRegion1)
                {
                    FormRegion1 formRegion1 = (FormRegion1)formRegion;
                    formRegion1.textBox1.Text = "Hello World";
                }
            }

        }